documentation/get-started/tutorials/get-started-qfc.en.md

```diff
@@ -230,6 +230,26 @@ Once configured, you can press the cloud button to open the synchronization dial
 
 Now you should see your project and files on [QFieldCloud](https://app.qfield.cloud/)
 
+### Activate Notifications for QFieldCloud changes
+
+1. Access the Settings of your account.
+2. Navigate to the Notifications section. Here, you can customize the frequency of notifications you wish to receive at the valid email address registered with your account.![Synchronize](../../assets/images/frequency_notifications_settings.png)
```

Comment:
What is the "valid" email address?

Reply:
Regardless of what exactly we check on code side, I wonder what value the term "valid" in this context will give to the reader of the documentation.

What I understand: a user has registered an email address with his account. This is checked by sending a validation email before a user is activated on the cloud. So in general we _trust_ users that their email address is valid. Referring to "the valid email address" suggests that there's something else like "an invalid email address" which to me sounds confusing without more context.

Reply:
Yes we should just cross out `valid`